Automattic/jetpack
# AI Client: add logo generator placeholder components

The Logo Generator modal is still up for a revisit. This PR tries to make the UX less quirky when it comes to components waiting for some process and suddenly popping in the DOM and making the content jumpy.

## Proposed changes:
Introduce isLoadingHistory to store
Introduce placeholder component for LogoPresenter
Introduce placeholder component for HistoryCarousel

These introductions should make the experience a bit more comfortable to the eye by setting visual expectations and avoiding content jumps due to components sudden render

### Other information:


## Jetpack product discussion

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Might be better to test side by side with/without the patch, here's a sample of what it looks like in both cases:


| Before | After
| - | -
| <video src="https://github.com/user-attachments/assets/bae713c2-c609-4649-ade5-eedc62fc6290" alt="before"/> | <video src="https://github.com/user-attachments/assets/8d0a092c-b5bd-49bf-b2c7-45189f76f1be" alt="after" />

Insert a Logo block and use the AI toolbar button to open the modal. As soon as the modal opens it will either load the history of logos (previously created for the site) or jump into generating a logo based on the site details.

On both cases, content shouldn't be jumpy, a placeholder should be in place for the logo presenter (main grey area) and the history carousel (collection of generated logos at the bottom of the modal).
